README.md

```diff
@@ -1,3 +1,66 @@
-# Mercurius Federation Fnfo
+# Mercurius Federation Info
 
-A Mercurius plugin that exports the structure of the federation.
\ No newline at end of file
+A Mercurius plugin that exports the structure of the federation.
+The plugin can be invoked by sending http `GET` request on route `/federation-schema`.
+The response object has this structure:
+
+```json
+{
+  "status": "OK",
+  "nodes": {
+    "node-1": { // the name of the federated node
+      "__schema": {...}
+    },
+    "node-2": {
+      "__schema": {...}
+    }
+  }
+}
+```
+
+The `__schema` object describes the configuration of the federated node.
+
+```json
+{
+      "__schema": {
+        "description": null,
+        "queryType": {
+          "name": "Query"
+        },
+        "mutationType": null,
+        "subscriptionType": null,
+        "types": [...],
+        "directives": [...]
+      }
+    }
+}
+```
```

Comment:
My comment below is probably in the wrong position and not clear.
I will add the docs for the additional fields we set in thee structure: esExtension, required, etc...